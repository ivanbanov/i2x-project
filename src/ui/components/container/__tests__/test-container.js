import React from 'react';
import renderer from 'react-test-renderer';
import Container from '../';

jest.mock('../styles.styl', () => ({}));

describe('Container', () => {
  it('should render properly', () => {
    const component = renderer.create(
      <Container>
        <div>Container</div>
      </Container>
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
