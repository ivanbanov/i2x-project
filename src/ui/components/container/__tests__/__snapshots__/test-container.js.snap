// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render properly 1`] = `
<div
  className={undefined}
>
  <div>
    Container
  </div>
</div>
`;
